cli.js

```diff
@@ -11,15 +12,53 @@ function _pickConfig (config) {
   /* eslint-enable */
 }
 
+const parser = argly
+  .createParser({
+    '--help -h': {
+      type: 'string',
+      description: 'Show this help message'
+    },
+    '--version -v': {
+      type: 'string',
+      description: 'Show the version number of mocha-puppeteer'
+    },
+    '--pattern -p *': {
+      type: 'string',
+      description: 'Pattern to run tests. Either a single file or glob pattern.'
+    }
+  })
+  .example('Test a single file: "mocha-puppeteer /foo/bar-test.js"')
+  .example('Test a series of files using a glob pattern: "mocha-puppeteer /foo/**/*-test.js"')
+  .validate(function (result) {
+    if (result.help) {
+      this.printUsage()
+    } else if (result.version) {
+      console.log(`mocha-puppeteer@${mochaPuppeteerPkgVersion}`)
+    } else if (!result.pattern) {
+      this.printUsage()
+    }
+  })
+  .onError(function (err) {
+    this.printUsage()
+    throw new Error('Error parsing command line args: ', err)
+  })
+
 module.exports = async function runCli () {
+  const {
+    pattern,
+    help,
+    version
+  } = parser.parse()
+
+  // Gracefully exit if either the "help" or "version" arguments are supplied
+  // of if the "pattern" argument is missing because it's required.
+  if (help || version || !pattern) return
```

Comment:
I thought about just calling `process.exit(0)` up in the `validate`, but if tests are added around this file, it will be difficult to test.

Reply:
Good call, stubbing `process.exit` would be a little dirty